tests/forte/data/readers/audio_reader_test.py

```diff
@@ -42,23 +45,27 @@ def initialize(self, resources: Resources, configs: Config):
         self._model = Wav2Vec2ForCTC.from_pretrained(pretrained_model)
 
     def _process(self, input_pack: DataPack):
+        ap = input_pack.get_payload_at(Modality.Audio, 0)
+        sample_rate = ap.sample_rate
+        audio_data = ap.cache
         required_sample_rate: int = 16000
-        if input_pack.sample_rate != required_sample_rate:
+        if sample_rate != required_sample_rate:
             raise ProcessFlowException(
                 f"A sample rate of {required_sample_rate} Hz is requied by the"
                 " pretrained model."
             )
 
         # tokenize
         input_values = self._tokenizer(
-            input_pack.audio, return_tensors="pt", padding="longest"
+            audio_data, return_tensors="pt", padding="longest"
         ).input_values  # Batch size 1
 
         # take argmax and decode
         transcription = self._tokenizer.batch_decode(
             argmax(self._model(input_values).logits, dim=-1)
         )
 
+        tp = TextPayload(input_pack, 0)
```

Comment:
This line might be redundant since `input_pack.set_text` will automatically create a `TextPayload`.